src/status_im/config.cljs

```diff
@@ -24,7 +24,7 @@
 (goog-define ALCHEMY_OPTIMISM_MAINNET_TOKEN "")
 (goog-define ALCHEMY_OPTIMISM_GOERLI_TOKEN "")
 (goog-define ALCHEMY_OPTIMISM_SEPOLIA_TOKEN "")
-(goog-define WALLET_CONNECT_PROJECT_ID "")
+(goog-define WALLET_CONNECT_PROJECT_ID "87815d72a81d739d2a7ce15c2cfdefb3")
```

Comment:
Probably shouldn't really be commited? Or does it actually matter at this stage?